src/components/CreatePost/Location.tsx

```diff
@@ -1,10 +1,27 @@
 import { LocationOnOutlined } from '@mui/icons-material';
 import { Box, IconButton, OutlinedInput } from '@mui/material';
+import { ChangeEvent, useState } from 'react';
+
+interface LocationProps {
+  readonly?: boolean;
+}
+
+const Location = ({ readonly }: LocationProps) => {
+  const [value, setValue] = useState('');
+  const onChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setValue(e.target.value);
+  };
 
-const Location = () => {
   return (
     <Box sx={locationBoxStyle}>
-      <OutlinedInput fullWidth placeholder='지역을 입력하세요' type='text' />
+      <OutlinedInput
+        fullWidth
+        placeholder='지역을 입력하세요'
+        type='text'
+        onChange={onChange}
+        readOnly={readonly && readonly}
```

Comment:
p3;

```suggestion
        readOnly={readonly}
```
readonly props 자체가 boolean 타입이라서 따로 논리 연산자를 사용하지 않아도 되겠네요!